examples/layer-browser/src/utils/grid-aggregator.js

```diff
@@ -1,12 +1,12 @@
-const R_EARTH = 6378;
+const R_EARTH = 6378000;
 
 /**
  * Aggregate points for the grid layer
  * @param {array} points
- * @param {number} worldUnitSize - unit size in kilometer
+ * @param {number} cellSize - cell size in meters
  * @returns {object} - grid data and cell dimension
  */
-export function pointsToWorldGrid(points, worldUnitSize) {
+export function pointsToWorldGrid(points, cellSize) {
```

Comment:
So you changed all worldUnitSize to cellSize here. Should we do the same in src/layers/core/grid-layer/grid-aggregator.js